collectors/o365/o365_collector.js

```diff
@@ -49,8 +49,10 @@ class O365Collector extends PawsCollector {
             startTs = moment().subtract(PARTIAL_WEEK_HOURS, 'hours').toISOString();
             console.info("O365000004 Start timestamp is more than 7 days in the past. This is not allowed in the MS managment API. setting the start time to 7 days in the past");
         }
-
-        if(moment().diff(startTs, 'hours') > 1){
+        if (process.env.paws_collection_interval && process.env.paws_collection_interval > 0) {
```

Comment:
I suggest we move this `if` statement into `pawsGetLogs()` and keep init function as is.